agentops/agentops.py

```diff
@@ -89,12 +89,12 @@ def wrapper(*args, **kwargs):
 
         return decorator
 
-    def start_session(self, tags: Optional[Dict[str, str]] = None):
+    def start_session(self, tags: Optional[List[str]] = None):
         """
         Start a new session for recording events.
 
         Args:
-            tags (Dict[str, str], optional): Tags that can be used for grouping or sorting later. Examples could be {"llm": "GPT-4"}.
+            tags (List[str], optional): Tags that can be used for grouping or sorting later. Examples could be ['GPT-4']}.
```

Comment:
nit: we're using double quotes everywhere else for the List example

Reply:
Gonna have to set up a linting protocol. We're inconsistent